pkg/serviceaccount/util.go

```diff
@@ -61,14 +65,6 @@ func InternalIsServiceAccountToken(secret *api.Secret, sa *api.ServiceAccount) b
 
 	name := secret.Annotations[api.ServiceAccountNameKey]
```

Comment:
why not just `return IsServiceAccountToken(secret, sa)` here no need for a new function.